WooCommerce/WooCommerceTests/Mocks/MockProductReviewFromNoteParcel.swift

```diff
@@ -0,0 +1,41 @@
+import Foundation
+import XCTest
+@testable import Yosemite
+
+
+final class MockProductReviewFromNoteParcel {
+    func parcel(metaSiteID: Int64 = 0) -> ProductReviewFromNoteParcel {
+        let metaAsData: Data = {
+            var ids = [String: Int64]()
+            ids[MetaContainer.Keys.site.rawValue] = metaSiteID
+
+            var metaAsJSON = [String: [String: Int64]]()
+            metaAsJSON[MetaContainer.Containers.ids.rawValue] = ids
+            do {
+                return try JSONEncoder().encode(metaAsJSON)
+            } catch {
+                XCTFail("Expected to convert MetaContainer JSON to Data. \(error)")
+                return Data()
+            }
+        }()
+
+        let note = Note(noteID: 1,
+                        hash: 0,
+                        read: false,
+                        icon: nil,
+                        noticon: nil,
+                        timestamp: "",
+                        type: "",
+                        subtype: nil,
+                        url: nil,
+                        title: nil,
+                        subject: Data(),
+                        header: Data(),
+                        body: Data(),
+                        meta: metaAsData)
+        let product = Product.fake()
+        let review = MockReviews().anonymousReview()
```

Comment:
We can change this to use `fake` too:

```suggestion
        let review = ProductReview.fake()
```

The test still passed when I replaced it. 🤷

Reply:
Nice! Updated in: https://github.com/woocommerce/woocommerce-ios/pull/5044/commits/15d1131bbe5e1c8d293066161c905502a76d48e5